README.md

```diff
@@ -122,12 +119,20 @@ Component supports dynamic incoming metadata - as soon as your query is in place
 
 ### LOOKUP BY PRIMARY KEY
 ![image](https://user-images.githubusercontent.com/40201204/43592505-5b6bbfe8-967e-11e8-845e-2ce8ac707357.png)
+
 The action will execute select query from a ``Table`` dropdown field, as criteria can be used only [PRIMARY KEY](https://en.wikipedia.org/wiki/Primary_key "PRIMARY KEY"). The action returns only one result (a primary key is unique).
 Checkbox ``Don't throw Error on an Empty Result`` allows to emit an empty response, otherwise you will get an error on empty response.
 #### Input fields description
 ![image](https://user-images.githubusercontent.com/40201204/43644579-f593d1c8-9737-11e8-9b97-ee9e575a19f7.png)
 As an input metadata you will get a Primary Key field to provide the data inside as a clause value.
 
+### UPSERT BY PRIMARY KEY
+![image](https://user-images.githubusercontent.com/16806832/44396836-d3880280-a546-11e8-8fbd-efc1fd089d93.png)
+The action will execute select query from a ``Tables`` dropdown field, as criteria can be used only [PRIMARY KEY](https://en.wikipedia.org/wiki/Primary_key "PRIMARY KEY"), if result is nullable, then action will execute insert query by PRIMARY KEY with specified field, else - action will execute update query by PRIMARY KEY with specified field. The action returns only one result row (a primary key is unique).
```

Comment:
There were changes in query builders, and now the action will execute Merge SQL-statement instead of pair lookup-upsert. This behaviour was implemented for preventing conflicts in the process with other sessions.